README.md

```diff
@@ -48,7 +50,17 @@ brew tap anchore/syft
 brew install syft
 ```
 
-**Note**: Currently, Syft is built only for macOS and Linux.
+### Nix
+
```

Comment:
Thanks for providing nix packaging --this is awesome! Can you add a note mentioning that this is community maintained? 

```suggestion
**Note**: nix packaging of Syft is community maintained
```

Reply:
ahh of course the homebrew release is a cask. I'll add that